dashboard/client/src/components/LogView/LogVirtualView.tsx

```diff
@@ -17,6 +16,8 @@ import "./github.css";
 import "./index.css";
 import { MAX_LINES_FOR_LOGS } from "../../service/log";
 
+lowlight.registerLanguage("prolog", prolog);
```

Comment:
curious what this is for?

Reply:
After lowlight 2.0, it introduced three import modes: `core`, `common` (default), and `all`. To prioritize simplicity and extensibility, we opted for the default `common` mode, which auto-registers 30+ languages. However, `prolog` is not included in the list. Therefore, we need to manually register it here.